docs/pages/setup/reference/config.mdx

```diff
@@ -35,8 +35,11 @@ teleport:
     # PID file for Teleport process
     #pid_file: /var/run/teleport.pid
 
-    # Invitation token used to join a cluster. it is not used on
-    # subsequent starts
+    # Invitation token or file path containing token used to join a cluster. it is not used on
+    # subsequent starts.  
+    # File path example:
+    # auth_token: /var/lib/teleport/tokenjoin
+    
```

Comment:
```suggestion
```
I think this was the extra line, since the other option-specific comments have no line between the comment and the option.